users/models.py

```diff
@@ -111,7 +111,7 @@ def create_superuser(self, username, email, password, **extra_fields):
         if extra_fields.get("is_superuser") is not True:
             raise ValueError("Superuser must have is_superuser=True.")
 
-        return self._create_user(username, email, password, **extra_fields)
+        return self._create_user(username, email, password=None, **extra_fields)
```

Comment:
Still need to be able to pass `password` because the django command to create superusers uses this method.